config/manager/image_config.yaml

```diff
@@ -3,7 +3,7 @@ kind: ImageConfig
 metadata:
   name: image-config
 images:
-  system-logger: "k8ssandra/system-logger:latest"
+  system-logger: "artifacts.k8ssandra.io/k8ssandra/images/system-logger:latest"
```

Comment:
Same here, won't work in local builds correctly.

Reply:
Also, if you wonder why to my previous question - the repository is different here than anything I would build. In the previous Makefile we build:

us-docker.pkg.dev/k8ssandra/images/system-logger:latest (which itself is problematic, but see another comment), but here we're calling ``artifacts.k8ssandra.io`` as prefix. So while your development env will popup, it's not actually loading the correct image, it's loading some historic one from the official repo, which is not what I'm actually developing.

Reply:
To be clear, `us-docker.pkg.dev` and `artifacts.k8ssandra.io` are the **same** repository. We want to _push_ all images to `us-docker.pkg.dev` and _pull_ from `artifacts.k8ssandra.io`.

Reply:
But that's not the same string - thus if you push there and load from another, you will not load from say local cache (like kind).